packages/experiments-realm/work-tracker.gts

```diff
@@ -625,3 +645,22 @@ function removeFileExtension(cardUrl: string) {
 function getComponent(cardOrField: BaseDef) {
   return cardOrField.constructor.getComponent(cardOrField);
 }
+
+// all search docs point to a relative link
+// we must derive the relative link based upon the code ref and the current realm
+function getRelativeUrl(
+  absoluteUrl: string, // absolute link from card model, eg http://localhost:4201/experiments/Project/51ca069b-1c55-4648-8654-bd82bf162f9d
+  codeRefToQuery: ResolvedCodeRef, // code ref to query eg {module: 'http://localhost:4201/experiments/productivity/task', name: 'Task'}
+  realmURL: URL, // realm url eg http://localhost:4201/experiments
+) {
+  let instanceDirectoryUrl = realmURL.href + codeRefToQuery.name + '/';
+  let relativeTo = relativeURL(
+    new URL(absoluteUrl),
+    new URL(instanceDirectoryUrl),
+    realmURL,
+  );
+  if (!relativeTo) {
+    throw new Error(`Missing relative url`);
+  }
+  return relativeTo;
+}
```

Comment:
I do not know why @field assignee has the indexed url in the search doc absolute whereas @field project has the indexed url in the search doc relative

Reply:
Yeah I came across this issue too. Sometimes they're absolute, sometimes relative. I think when it's a different realm, it will have to be an absolute url.